contracts/scripts/test/smokeTest/mintRedeemTest.js

```diff
@@ -0,0 +1,94 @@
+const { fund, mint } = require('../../../tasks/account');
```

Comment:
As opposed to putting those tests under contracts/scripts/test/smokeTest, I wonder if we should less deep structure to make them more easily discoverable. And put them under contracts/smoke. I don't feel strongly. Just providing a possible alternative.

Reply:
Good point moving it to `smoke`